### ISSUE TITLE
WebSocket closed with reason: (1006).

### ISSUE THREAD

**Issue by alkar22** [chathub-dev/chathub#823]
**WebSocket closed with reason: (1006).**
Most of the AI Tools cannot be used, when try it throws error messages. 
exaample , Bing - it throws "WebSocket closed with reason: (1006)" 
please fix them

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Browser [e.g. chrome, brave]

**Comment by jinma-wish** [comment]
with the same situation

**Comment by wong2** [comment]
Does retry help?

**Comment by jinma-wish** [comment]
no

**Comment by 3L1AS** [comment]
I get the same but only on Llama 2: WebSocket closed with reason: (1006).

**Comment by sunshineynwa** [comment]
Any luck on this? I get the same with Llama 2: WebSocket closed with reason:  (1006).

**Comment by XZ1314XZ** [comment]
![image.png](https://pic4.58cdn.com.cn/nowater/webim/big/n_v256a5276cfdb6471c99beed08227ff9b8.png)

**Comment by wong2** [comment]
@XZ1314XZ Please open https://chat.lmsys.org and see if you can chat normally on the website

**Comment by XZ1314XZ** [comment]
@wong2  Alright, I successfully resolved the issue. After successfully accessing [https://chat.lmsys.org/] once, the plugin can now be used normally. I guess it might have triggered Cloudflare's security verification or the user agreement for the first time.

**Comment by 3L1AS** [comment]
Works for me. Also at chat.lmsys.org
![Screenshot 2023-11-06 104533](https://github.com/chathub-dev/chathub/assets/19346927/d1c81442-2d1f-471a-b8e6-b1c2dd48c918)

**Comment by 3L1AS** [comment]
> @wong2 Alright, I successfully resolved the issue. After successfully accessing [https://chat.lmsys.org/] once, the plugin can now be used normally. I guess it might have triggered Cloudflare's security verification or the user agreement for the first time.

Ah okay, this makes sense then.

**Comment by wong2** [comment]
Yes, I just noticed that LMSYS added a Cloudflare protection. I'll update the code to suggest users visit lmsys website in this case.
